Code/Engine/Foundation/Reflection/Implementation/PropertyAttributes.h

```diff
@@ -336,6 +336,31 @@ class EZ_FOUNDATION_DLL ezConstrainPointerAttribute : public ezPropertyAttribute
   ezUntrackedString m_sConstantValueProperty;
 };
 
+/// \brief Defines how a reference set by ezFileBrowserAttribute is treated.
+struct ezDependencyFlags
+{
+  typedef ezUInt8 StorageType;
+
+  enum Enum
+  {
+    None = 0,
+    Thumbnail = EZ_BIT(0), ///< This reference is a dependency to generating a thumbnail.
+    Transform = EZ_BIT(1), ///< This reference is a dependency to transforming this asset.
+    Package = EZ_BIT(2),   ///< This reference is needs to be packaged as it is used at runtime by this asset.
```

Comment:
These are quite important to understand to use correctly, I think we should document this a bit more thoroughly. Also the None case. Maybe with a concrete example for each.

Reply:
I've added a few examples to each element and to the class itself.